integrations/pagerduty/integration.py

```diff
@@ -88,6 +88,11 @@ class PagerdutySelector(Selector):
         api_query_params: PagerdutyIncidentAPIQueryParams | None = Field(
             alias="apiQueryParams"
         )
+        enable_incident_analytics: bool = Field(
+            default=False,
+            description="If set to true, will ingest incident analytics data to Port. Default value is false",
+            alias="enableIncidentAnalytics",
+        )
```

Comment:
Its under selector so I feel like there is no need to add the "enable" verb as we are saying "what to select" under the selector 🤔 
Let me know if that makes sense for you
```suggestion
        incident_analytics: bool = Field(
            default=False,
            description="If set to true, will ingest incident analytics data to Port. Default value is false",
            alias="IncidentAnalytics",
        )
```